### DISCUSSION TITLE
커스텀 어태치먼트 질문

### DISCUSSION THREAD

**Discussion by cjh980402** [storycraft/node-kakao#158] [Q&A] [answered]
버전 4에서는 버전 3에서 있었던 커스텀 어태치먼트는 지원이 안되나요?

**Comment by storycraft** [comment, marked as answer]
SEARCH 타입과 CUSTOM 타입 어태치먼트의 타입스크립트 타이핑은 아직 지원이 되고 있지 않습니다. 단 어태치먼트 타이핑과 관계없이 받거나 전송하는 것은 가능합니다

**Reply by cjh980402** [reply]
답변 감사합니다. 버전 3의 AttachmentTemplate은 버전 4에서의 attachment쪽과 관련 있겠죠?

**Reply by storycraft** [reply]
버전 3의 AttachmentTemplate은 attachment wrapper 객체들을 보내는 용도였지만 버전 4에서는 attachment의 타이핑만 지원하고 챗 타입에 따른 wrapper를 따로 제공하지 않고 있습니다.
